features/step_definitions/challege_gift_exchange_steps.rb

```diff
@@ -111,6 +111,17 @@
   step %{I open signups for "#{title}"}
 end
 
+Given /^the open gift exchange "([^\"]*)" has overlapping tags$/ do |challengename|
+  step %{I am logged in as "mod1"}
+  step "I have overlapping challenge tags setup"
+  step %{I set up the collection "#{challengename}" with name "#{challengename.gsub(/[^\w]/, '_')}"}
```

Comment:
I think this can be simplified to
```suggestion
  step %{I set up the collection "#{challengename}"
```
because of the `blank?` bit in https://github.com/otwcode/otwarchive/blob/4e1a23efe1f0ea792047c5405d91c9bc06d77c85/features/step_definitions/collection_steps.rb#L120